src/std/iter.ss

```diff
@@ -139,23 +139,26 @@ package: std
   ((count start) (iter-in-iota start count 1))
   ((count start step) (iter-in-iota start count step)))
 
-(def (iter-in-range start end step)
+(def (iter-in-range start end (step 1) (cmp <))
```

Comment:
Having default values here makes `in-range` a little less readable IMO. See next comment.